packages/yaml/src/index.ts

```diff
@@ -0,0 +1,129 @@
+import { type Service } from '@volar/language-service';
+import { type TextDocument } from 'vscode-languageserver-textdocument';
+import { type LanguageSettings, type LanguageService } from 'yaml-language-server';
+import { getLanguageService } from 'yaml-language-server/lib/umd/languageservice/yamlLanguageService.js';
```

Comment:
This is a good way to avoid downstream bundling hack! But since this is inconsistent with other services, I will temporarily change it to `import from 'yaml-language-server'` and change to import from umd path for all services in a separate PR, so we can retrace the reason.